micro-rdk/Cargo.toml

```diff
@@ -19,7 +19,7 @@ default = ["builtin-components", "data"]
 binstart = ["esp-idf-svc/binstart"]
 libstart = ["esp-idf-svc/libstart"]
 builtin-components = []
-camera = []
+camera = ["builtin-components"]
```

Comment:
I'm not sure I see why. Also wondering: if !builtin-components should you even get the camera component? Maybe you need to ask for both `camera` (to get support) and `builtin-components` (to get the actual camera component)? After all, you might want to build your own camera component in a module but not use ours?

Reply:
following offline conversation
- camera: enable traits
- camera && builtin: enable traits, fake_camera, esp32camera (on esp32)

Reply:
@acmorrow ok to resolve?